tests/acceptance/features/webUIUpload/upload.feature

```diff
@@ -86,7 +86,7 @@ Feature: File Upload
     And file "new-lorem.txt" should be listed on the webUI
     And as "Alice" the content of "simple-folder/new-lorem.txt" should be the same as the content of local file "new-lorem.txt"
 
-  @smokeTest
+  @smokeTest @disablePreviews
```

Comment:
adding `@disablePreviews` makes the test pass. It seems that `@disablePreviews` tag reduces locking issue